src/events/components/ImageView/EventColumn.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { INewEvent } from 'events/models/Event';
+import { EventTypeEnum } from '../../models/Event';
+import style from './image.less';
+import LargeEvent from './LargeEvent';
+import LargeEventPlaceholder from './LargeEventPlaceholder';
+import SmallEventColumn from './SmallEvent';
+
+export interface IEventColumn {
```

Comment:
I would like to have it named as props in the interface name as well.
I think every other interface in the app describes it that way, and it is nice to keep it consistent :smile: